orchestra/communication/staffing.py

```diff
@@ -141,6 +141,24 @@ def _exclude_inactive_staffbot_requests(request_queryset):
             .exclude(task__status=Task.Status.ABORTED))
 
 
+def _get_worker_allowed_hours():
+    today = timezone.now().date()
+    today_abbreviation = ['mon', 'tues', 'wed', 'thurs', 'fri', 'sat', 'sun'][
+        today.weekday()]
+    return {
+        availability.worker.id:
+        min(
```

Comment:
add a comment explaining the two things we're min-ning here?

Reply:
Done! TY!